GoogleSignIn/Sources/GIDSignInInternalOptions.h

```diff
@@ -51,12 +54,14 @@ NS_ASSUME_NONNULL_BEGIN
 
 /// Creates the default options.
 + (instancetype)defaultOptionsWithConfiguration:(nullable GIDConfiguration *)configuration
-                       presentingViewController:(nullable UIViewController *)presentingViewController
+                       presentingViewController:
+                           (nullable UIViewController *)presentingViewController
                                       loginHint:(nullable NSString *)loginHint
-                                       callback:(GIDSignInCallback)callback;
+                                       callback:(nullable GIDSignInCallback)callback;
 
 /// Creates the options to sign in silently.
-+ (instancetype)silentOptionsWithCallback:(GIDSignInCallback)callback;
++ (instancetype)silentOptionsRefreshAccessToken:(BOOL)refreshAccessToken
+                                       callback:(nullable GIDSignInCallback)callback;
```

Comment:
It may be slightly easier to read if you keep the existing +silientOptionsWithCallback: method and add a new +silientOptionsNoRefresh method.

Reply:
Done.